lib/wccp/WccpEndPoint.cc

```diff
@@ -373,7 +373,6 @@ CacheImpl::defineServiceGroup(ServiceGroup const &svc, ServiceGroup::Result *res
   if (spot == m_groups.end()) { // not defined
     group        = &(m_groups[svc_id]);
     group->m_svc = svc;
-    memset(&group->m_id, 0, sizeof(group->m_id));
```

Comment:
I looked at this a long and time I don't know why I did that - it seems useless because it's in a `std::map` and it should be constructed which should zero out all the elements. Although this shouldn't break anything, gcc 8 didn't like it so since it's useless I took it out.